src/server/pkg/collection/collection.go

```diff
@@ -32,7 +33,7 @@ const (
 var (
 	// ErrNotClaimed is an error used to indicate that a different requester beat
 	// the current requester to a key claim.
-	ErrNotClaimed = fmt.Errorf("NOT_CLAIMED")
+	ErrNotClaimed = errors.Errorf("NOT_CLAIMED")
```

Comment:
Yeah, this is a general problem I encountered. Error constants like this don't work well in this model. The stack information isn't useful, and you can't wrap them because then it breaks the comparisons. Once we migrate all of the comparisons to `errors.Is` though we can start wrapping them when they're first returned and that should work well. Maybe we don't want to create the stacks here since they're not very useful, I wanted to just go through and remove all `fmt.Errorf` calls from the codebase, and am even planning to add a linter check to make sure people don't introduce new ones. But this is what linter exceptions are for. Also the stack trace for an error like this is going to be 32 bytes, which given that you only ever have one copy of this error probably isn't a big deal.